4. Projects/1. CLI/Problem/Stage 1/Step 2/src/db.rs

```diff
@@ -108,10 +91,7 @@ mod tests {
             let write_result = db.write_db(&state);
             let read_result = db.read_db().unwrap();
 
-            remove_file(file_path).unwrap();
-
             assert_eq!(write_result.is_ok(), true);
-            // TODO: fix this error by deriving the appropriate traits for DBState
```

Comment:
Why did you remove this comment?

Reply:
Sorry, there were a lot of files that shared the same tests, I copy and pasted the relevant tests and didn't realize this was in that stage!